### ISSUE TITLE
ELI5 how to use this

### ISSUE THREAD

**Issue by wlfeh** [AustynKoky/TicketScoop#1]
Can you explain like I'm 5 on how to use this? How do I obtain the AUTH token? Is this only for mobile devices? Thanks brother.

**Comment by KokyCode** [comment]
Hey! The AUTH token is your bearer token which can be found in the API when doing requests such as adding to cart. You should be logged in when doing this, but it will require you to scrape the data from the web or mobile app.

Hope this helps.
